### PR TITLE
chore: use posix compatible command to check if command exists on OS

### PR DESCRIPTION
**Pull request by sanderblue**

### CODE REVIEW

File: recipes/newrelic/infrastructure/amazonlinux.yml [hunk 1]
```diff
@@ -142,6 +142,5 @@ postInstall:
       ⚙️  The Infrastructure Agent configuration file can be found in /etc/newrelic-infra.yml
       Edit this file to make changes or configure advanced features for the agent. See the docs for options:
       https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings
-      
-      Note: Process monitoring has been enabled by default - all other config options are left to the user.
 
+      Note: Process monitoring has been enabled by default - all other config options are left to the user.
```
Comment (jbeveland27) [inline recipes/newrelic/infrastructure/amazonlinux.yml:146]: I'd be careful about removing the indentation on this empty line. Might be invalid yaml without it.

Comment (sanderblue) [reply recipes/newrelic/infrastructure/amazonlinux.yml:146]: My auto formatter did this

Comment (sanderblue) [reply recipes/newrelic/infrastructure/amazonlinux.yml:146]: And yep, I think it's causing an issue. Thanks

**Review by Julien4218 — APPROVED** [review]

**Review by jbeveland27 — APPROVED** [review]
LGTM 🚀

**Pull request merged**